tests/features/post.feature

```diff
@@ -1,6 +1,12 @@
 Feature: Post
   Test the importing an post
 
+  Scenario: Article has correct number of images
```

Comment:
Just a minor thing: Can you say its "image widgets" rather than just "images"? As there may be other ways of adding images in to the article other than the Image Widget, such as a number of hero images, and List Items which include images.